compiler/luci/pass/include/luci/Pass/FakeQuantizationPass.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __LUCI_FAKE_QUANTIZATION_PASS_H__
+#define __LUCI_FAKE_QUANTIZATION_PASS_H__
+
+#include <logo/Pass.h>
+
+namespace luci
+{
+
+/**
+ * @brief  Class to convert a quantized model to a fake-quantized fp32 model.
+ */
+struct FakeQuantizationPass final : public logo::Pass
```

Comment:
There is `FakeQuant` Op but this pass inserts `Quantize` and `Dequantize` Ops.
As I understand these two ops will act like `FakeQuant` but this gives a little confusion. 
How about something like `InsertQuantDequantPass` ?

Reply:
Hmm.. This pass does fake quantization, and insertion of quant/dequant is just a part of its process. `InsertQuantDequantPass` may not correctly describe what it does.

How about `ConvertToFakeQuantizedModelPass`?

Reply:
`ConvertToFakeQuantizedModelPass` looks OK :)